CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * The old `libgit2` code path for fetches and pushes has been removed,
   and the `git.subprocess` setting along with it.
 
+* Change IDs are now stored in the Git commit by default. Disable by
+  setting `git.write-change-id-header` to `false`. Please let us know if you do.
```

Comment:
Does this count as a breaking change? Maybe it should be a release highlight instead (possibly with mention of the consequence of being able to sync change IDs, and caveats around forge rebases)?

Reply:
I think you're right. I've moved it to "Release highlights". PTAL at the wording.

Reply:
LGTM. I would be tempted to explicitly mention that some forges will clobber it, but maybe that kind of thing is better kept out of release notes since it duplicates things that need to be in regular docs anyway. (I just anticipate “I upgraded and it doesn’t work with GitLab” type issues.)

Reply:
Done. I mentioned GitLab and `git rebase` as examples of where change ids are not preserved.